pkg/controllers/scheduler/framework/plugins/rsp/rsp.go

```diff
@@ -161,6 +166,7 @@ func (pl *ClusterCapacityWeight) ReplicaScheduling(
 		ExtractClusterNames(clusters),
 		currentReplicas,
 		estimatedCapacity,
+		limitedCapacity,
```

Comment:
Can we combine `estimatedCapacity` and `limitedCapacity` by taking the minimum for each element before calling `planner.Plan`? The planner is already very complicated and let's not add to its complexity.

Reply:
I am afraid we cannot. Because there are a little differences. For example, limitedCapacity does not have overflow logic.